py/selenium/webdriver/support/expected_conditions.py

```diff
@@ -124,6 +125,27 @@ def __call__(self, driver):
         return [element for element in _find_elements(driver, self.locator) if _element_if_visible(element)]
 
 
+class visibility_of_all_elements_located(object):
+    """ An expectation for checking that all elements are present on the DOM of a
+    page and visible. Visibility means that the elements are not only displayed
+    but also has a height and width that is greater than 0.
+    locator - used to find the elements
+    returns the list of WebElements once they are located and visible
+    """
+    def __init__(self, locator):
+        self.locator = locator
+
+    def __call__(self, driver):
+        try:
+            elements = _find_elements(driver, self.locator)
+            for element in elements:
+                if not _element_if_visible(element):
```

Comment:
@BeyondEvil  just push your changes to the same remote branch again... the PR will get updated automagically.